lua/telescope/pickers/layout_strategies.lua

```diff
@@ -806,13 +807,22 @@ layout_strategies.bottom_pane = make_documented_layout(
     results.height = height - prompt.height - (2 * bs)
     preview.height = results.height - bs
 
+    local width_opt = layout_config.width
+    local width = resolve.resolve_width(width_opt)(self, max_columns, max_lines)
+
     -- Width
     prompt.width = max_columns - 2 * bs
-    -- TODO(l-kershaw): add a preview_cutoff option
-    if self.previewer then
-      -- TODO(l-kershaw): make configurable
-      results.width = math.floor(max_columns / 2) - 2 * bs
-      preview.width = max_columns - results.width - 4 * bs
+    local w_space
```

Comment:
We will need to add `width` as a local variable here

Reply:
just `local width`or wym?
width is added like 4 lines above